### ISSUE TITLE
Cinnamon Screen Reader Accessibility

### ISSUE THREAD

**Issue by PlanetLinux98** [linuxmint/cinnamon#8462]
```
 * Cinnamon version 4
 * Distribution - Mint 19.1
 * Graphics hardware *and* driver used - Intel HD Graphics w/ opensource driver
 * 32 or 64 bit - 64-bit
 ```

**Issue**
The Screen Reader (which is Orca) has numerous issues with Mint 19.1 Cinnamon. Firstly, the typical shortcut toggle of Alt+S is not added by default. When the Screen Reader is turned on, the system gets incredibly sluggish for a few minutes. Also, most Cinnamon elements such as the main menu or panel elements do not read aloud when navigating through them.

**Steps to reproduce**
Navigate to Accessibility settings and enable the Screen Reader. Then access the main menu with the Super key and attempt to navigate through it with the arrow keys.

**Expected behaviour**
When the Screen Reader is first opened, the system lags and responds sluggishly. When attempting to navigate through the menu or various other Cinnamon elements, Orca screen reader does not read these.

**Other information**
I realize some of this could be issues for the Orca developer, but these are problems specific to Cinnamon, so it seems that some work needs to be done on the Mint side of things to make it accessible for everyone, including those that have little or no vision. (Included zoom features are great by the way)

**Comment by christophe-g** [comment]
Facing similar issues with Mint 19.3. 

Orca version in apt repository and installed in mint is old (3.28.9), which can be causing the issue. 

Reported on Orca bug tracker https://gitlab.gnome.org/GNOME/orca/-/issues/115 but seems downstream (mint/ubuntu) have some responsibility in this.

**Comment by goebbe** [comment]
In Linux Mint 21 Cinnamon Edition, the accessibility issues are still present:
- The standard shortcut for the ORCA screen reader:  Super+Alt+S   (to turn on the screenreader) does not work. 
  (Visual impaired (blind) people have a hard time to navigate to the accessibility screen to turn the screenreader on.) 
- Cinnamon Menu and panel seem not support the screenreader.

**Comment by christophe-g** [comment]
Thanks @goebbe - I can confirm it does not work on my machine, with a fresh update to Mint 21.

Reported a new bug here: https://github.com/linuxmint/cinnamon/issues/11146

**Comment by gigino1987** [comment]
Hi, I'm blind user and interact with pc via screen reader. I recently tried Cinnamon with Orca and there is some problem with applications menu and settings panel.
When I open apps menu with Super key I can't select any item with arrow keys but I must enter text on search bar and navigate with flat review to reach the whished app.
Same situation in settings panel. Please solve it because Cinnamon is best solution found.

**Comment by RonnyG2121** [comment]
I tried Linux Mint, and it works great with Mate. With Cinnamon, it's a big problem.
